### PR TITLE
Tests for mapmarkers and plots

### PR DESCRIPTION
**Pull request by tenrowd**
Began tests for map markers and plots and removed unused code from plots with its error.

### CODE REVIEW

File: tests/test_plots.py [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+    results = {"test_value": 1, "another_one": 0, "one_more": 3}
+    plots.create_pie_chart(results=results, suptitle=f"Test value")
+    plots.save_pie_chart(
+        relative_path=PlotsTestDefaultValues.PlotSubDirectory,
+        filename=f"{PlotsTestDefaultValues.FileName}",
+    )
+    assert PlotsTestDefaultValues.PathToFile.is_dir()
+    assert PlotsTestDefaultValues.PathToFile.exists()
+    assert PlotsTestDefaultValues.PathWithFile.exists()
+    assert PlotsTestDefaultValues.PathWithFile.stat().st_size == 72997
```
[the remark this answers is not in the document]
Comment (manmolecular) [reply tests/test_plots.py]: In this case, maybe we can calculate the file hash? For example, md5 will be ok I think.

Take a quick look, it's not the best way but you will get the basic idea: https://stackoverflow.com/questions/16874598/how-do-i-calculate-the-md5-checksum-of-a-file-in-python

File: tests/test_plots.py [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+    """
+    plots = GrinderPlots()
+    results = {"test": 1}
+    with raises(GrinderPlotsSavePieChartError) as create_error:
+        plots.create_pie_chart(results=results, suptitle=f"Test value")
+        plots.save_pie_chart(
+            relative_path=PlotsTestDefaultValues.PlotSubDirectory, filename=None
+        )
+    assert "expected str, bytes or os.PathLike object" in str(create_error.value)
+    PlotsTestDefaultValues.PathToFile.rmdir()
```
[the remark this answers is not in the document]
Comment (manmolecular) [reply tests/test_plots.py]: Check this out: https://docs.python.org/3/library/shutil.html#shutil.rmtree

File: tests/test_plots.py [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+    """
+    plots = GrinderPlots()
+    results = {}
+    plots.create_pie_chart(results=results, suptitle=f"Test value")
+    plots.save_pie_chart(
+        relative_path=PlotsTestDefaultValues.PlotSubDirectory,
+        filename=f"{PlotsTestDefaultValues.FileName}",
+    )
+    assert plots.results_figure_id == 0
+    assert not PlotsTestDefaultValues.PathWithFile.exists()
```
[the remark this answers is not in the document]
Comment (manmolecular) [reply tests/test_plots.py]: I think that this error can be the result of the previous one, related to directory removing (look at the answer with rmdir-shutil-rmtree)

File: tests/test_plots.py [hunk 4]
```diff
@@ -0,0 +1,131 @@
...
+
+    PlotsDirectory: Path = (
+        Path(".")
+        .joinpath(DefaultValues.RESULTS_DIRECTORY)
+        .joinpath(DefaultValues.PNG_RESULTS_DIRECTORY)
+    )
+    PlotSubDirectory: str = "test_plots"
+    PathToFile: Path = PlotsDirectory.joinpath(PlotSubDirectory)
+    FileName: str = "test_plots.png"
+    PathWithFile: Path = PathToFile.joinpath(FileName)
```
Comment (manmolecular) [inline tests/test_plots.py]: Same notes about CamelCase here, see previous comments from `test_mapmarkers.py`

File: tests/test_mapmarkers.py [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+
+class MapTestDefaultValues:
+    """
+    Needed Paths for test
+    """
+
+    MapDirectory: str = "tests/test_data/test_mapmarkers"
+    PathToTestMapDirectory: Path = Path(".").joinpath(MapDirectory)
+    PathToFile: Path = PathToTestMapDirectory.joinpath("static").joinpath("data")
+    PathWithFile: Path = PathToFile.joinpath("markers.json")
```
[the remark this answers is not in the document]
Comment (tenrowd) [reply tests/test_mapmarkers.py]: I'll do my best not to forget about the usage of these styles.

**Pull request merged**